marimo/_save/save.py

```diff
@@ -921,18 +921,12 @@ def my_expensive_function_cached_in_a_certain_location():
             `__marimo__/cache` in the directory of the notebook file
         method: the serialization method to use, current options are "json",
             and "pickle" (default).
-        pin_modules: if True, the cache will be invalidated if module versions
```

Comment:
There was a formatting change for docs strings made recently- but if this looks better then great:

https://github.com/marimo-team/marimo/pull/5750/files